modules/raft/src/main/java/org/apache/ignite/raft/jraft/entity/EnumOutter.java

```diff
@@ -77,24 +74,17 @@ public static EntryType valueOf(int value) {
             }
         }
 
-        /**
-        * Returns the enumerated value from its ordinal.
-        *
-        * @param ordinal Ordinal of enumeration constant.
-        * @throws IllegalArgumentException If no enumeration constant by ordinal.
-        */
-        public static EntryType fromOrdinal(int ordinal) throws IllegalArgumentException {
-            if (ordinal < 0 || ordinal >= VALUES.length) {
-                throw new IllegalArgumentException("No enum constant from ordinal: " + ordinal);
-            }
-
-            return VALUES[ordinal];
-        }
-
         private final int value;
 
-        private EntryType(int value) {
+        private final int transferableId;
```

Comment:
Do we actually need `value` and `transferableId` be separate things?

Reply:
I don't have an answer, I can merge them, what do you think?

Reply:
The existing value is used by `V1Decoder`, so this value has the same meaning as our `transferableId` and should never be changed. I think we can just reuse it as `transferableId()`.

Reply:
Ok